config/webpack/webpack.config.js

```diff
@@ -9,30 +9,25 @@ const TerserPlugin = require('terser-webpack-plugin');
 const merge = require('webpack-merge');
 const DuplicatePackageCheckerPlugin = require('@cerner/duplicate-package-checker-webpack-plugin');
 const aggregateTranslations = require('terra-aggregate-translations');
-const ThemeAggregator = require('../../scripts/aggregate-themes/theme-aggregator');
+const webpackEntries = require('./webpack.entries');
 
 const webpackConfig = (options, env, argv) => {
   const {
     rootPath,
     resolveModules,
-    themeFile,
     staticOptions,
   } = options;
 
   const production = argv.p;
-  let filename = production ? '[name]-[chunkhash]' : '[name]';
-  filename = argv['output-filename'] || filename;
+  const fileNameStategy = production ? '[name]-[chunkhash]' : '[name]';
+  const chunkFilename = argv['output-chunk-filename'] || fileNameStategy;
+  const filename = argv['output-filename'] || fileNameStategy;
```

Comment:
Why do we need to customize this? I'm concerned with how complicated this webpack config has grown.